### ISSUE TITLE
[Project] Swiss Tax Map

### ISSUE THREAD

**Issue by xeophin** [jsoma/data-studio-projects#182]
# Pitch

## Summary

Work called, and they want a data visualisation from last year updated with the most recent data!

This is a map of Switzerland, showing the tax burden in each municipality for various relationship models and incomes.

## Details

**Possible headline(s):** 

**Data set(s):** 
The table has been provided by the Federal Tax Administration

**Code repository:** 
https://gitlab.com/ta-interaktiv/content/2018-06-steuerkarte
	
**Possible problems/fears/questions:**
How can this visualisation be improved and be more helpful to the reader? 

## Work so far

This is [last year's version](https://interaktiv.tagesanzeiger.ch/2017/steuerbelastung/).

<img width="1154" alt="screen shot 2018-07-17 at 08 12 36" src="https://user-images.githubusercontent.com/658596/42816556-3dcc0048-8999-11e8-96a6-0c82278fe7b3.png">

I then [re-used the visualisation for another project](https://interaktiv.tagesanzeiger.ch/2017/mietpreis-karte/) and massively improved on performance.

I'm currently in the process of adapting the new and improved version back to the data model of the tax map.

## Checklist

This checklist must be completed before you submit your draft.

- [ ] I have already spent time with my data set, opening it, exploring it, etc
- [ ] I have created a "DIARY.md" file to save links and list all of the terrible, no good problems I come across
- [ ] My issue links to my data set(s)
- [ ] My issue links to my code repository
- [ ] My issue explains what I'd like to explore in the data set
- [ ] My issue includes images - either inspiration or what I've done so far

**Comment by xeophin** [comment]
# Update
	
## Your project content: images/words/etc
	
The new version is [now online and has the new data, but the old text](https://interaktiv.tagesanzeiger.ch/2018/steuerbelastung/). It looks like this:

<img width="777" alt="screen shot 2018-07-19 at 15 16 15" src="https://user-images.githubusercontent.com/658596/42965229-61336c36-8b67-11e8-969f-282df2b8a764.png">

## Any changes in direction or topic?
  
None so far
	
## Problems/Questions

- Presentation needs to be improved (municipality borders, canton borders, icons)
- There is a bug in Zürich displaying city districts instead of the whole municipality
- Search function and ranking list is still missing


## Checklist
	
- [ ] I have included my visuals
- [ ] I have filled out the sections above
- [ ] I have been updating my DIARY.md with details about my process
- [ ] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by playfairbot** [comment]
Hi! I'm a little robot, let's take a peek.

You need some feedback, let me summon @ElinaMak, @cfelke, @maxarvid for you

**Comment by cfelke** [comment]
Haha, this is a tough one, I mean look at this map!
The only thing I'm not sure about is your choice of colours, the pink is fine but I have my doubts about the neon green. Out of curiosity, why did you decide to go with two colors instead of one displaying various levels of opacity? And why does your range of taxation end at 32% while it seems that your data ranges from 2% to 14%?

**Comment by maxarvid** [comment]
Yikes, not much to add here except that the default is a pre taxation income of 150,000 😭.

I like the colour gradation, but have problems with the sizing. I realise that this amount of detail requires a large map, but it requires a lot of scrolling on the standard 13 inch screen. 

Et il y a un certain manque d'un traduction française...

**Comment by xeophin** [comment]
@cfelke: The colours have been taken over from a previous project. I agree that it could just have been one hue, it is certainly worth discussing. In this case it has been decided that red and green as "where to leave and where to go" was somewhat appropriate.

The scale is designed to show you the range of all available data, giving you an overview of all possible  values for all relationships and all income levels. In the best case you should be able how you compare to other people and how much of a difference it would actually make to move.

@maxarvid Yes, size is a problem, but due to time constraints I won't be able to implement a zoom function. And yes, there is no French translation, but ... this could happen if 24heures/Tribune de Geneve are interested in bringing this standalone to the French speaking part of Switzerland.

**Comment by xeophin** [comment]
# Final

## Project visuals/text

![screenshot_2018-07-26 hier finden sie ihr steuerparadies](https://user-images.githubusercontent.com/658596/43280829-f5994396-90df-11e8-904e-948abcb0ded2.png)

## Details

**Headline:** Hier finden Sie Ihr Steuerparadies (Here you will find your tax haven)

**Published website version:** https://interaktiv.tagesanzeiger.ch/2018/steuerbelastung/

**Code repository:** https://gitlab.com/ta-interaktiv/content/2018-06-steuerkarte

**Final data set(s):** 

## What did you find to be the most difficult part of this project?

Getting it done on time, collaborating across time zones and while supposedly doing LEDE ... 😬 

## Are you satisfied with what you produced? Is there anything you would like to change or improve?

Fairly satisfied – the performance is where it should be, so it was worth backporting the project to this data set.

- The tool tips could use some icon love, but I didn't have enough time to implement that
- The map could use a zoom function, but this was out of scope at this time
- The code weighs in at 2.5MB ... which is very, very large. There could be some improvements there, but again, time was lacking.

## Checklist

- [ ] I have included my visuals
- [ ] I have posted my project to my project website
- [ ] I have been updating my DIARY.md with details about my process
- [ ] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by sarahslo** [comment]
i'm going to weigh in on color here since you're on the fence about it and looking to improve the project. generally one moves to 2 colors in a color ramp when you are showing a positive or a negative, or two groups. the idea of ramps is to show the heat of the change, so the more intense the color, the more intense the number.

i'm not sure why your data are broken down so finely, into changes of 2%. that almost means you have to use more than 1 color, because in general, if you use more than 5 colors in a ramp it becomes too hard for readers to distinguish between them.

so if it's important to break these data down so finely i wonder if you want to bracket them into different groups? not understanding the data entirely - are these tax brackets?

or can you smooth the data so we can have 1 color, use increments of 5%?

**Comment by Palarisk** [comment]
This is absolutely mega intresting work! I spent ages exploring the one from last year (trying to find out who pays the most taxes and found it to be rich retired people)

I actually really like that you have to different colors here, it makes it easier to spot the tax havens.

One thing I would change next year: I would add a direct answer to the question "where is your tax haven"? I mean, when I choose my Berechnungsmodell and Bruttoeinkommen, it would be nice to get the answer not only on map but a highlighter/popup/separate column/something telling that "Your tax haven is XXXX". I think that already exists in the Gemeinde-chart down right, so maybe just add headline to that to make it clear what it stands for?
